### PR TITLE
Update Information that how to build on Linux

### PR DESCRIPTION
**Pull request by DevSDK**
Osu is multiplatform project.
So, I think that we need more explain build method.
And, I find very important thing that the openTK on Linux work with sdl2 library. 
If sdl2 library not install on system, osu never launch GUI system.
Im Added some text that How can build osu porject.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,36 @@ This is still heavily under development and is not intended for end-user use. Th
...
+
+### Build on ubuntu with commend line
+
+**1. clone the proejct**
+
+```git clone https://github.com/ppy/osu --recurse-submodules```
+
+**2. Install require packages**
+
+```sudo apt-get install nuget msbuild mono-complete libsdl2-dev```
```
Comment (peppy) [inline README.md]: this is out of scope

File: README.md [hunk 2]
```diff
@@ -11,6 +11,36 @@ This is still heavily under development and is not intended for end-user use. Th
...
+
+**3. Install nuget submodules**
```
Comment (peppy) [inline README.md]: nuget packages, not submodules.

File: README.md [hunk 3]
```diff
@@ -11,6 +11,36 @@ This is still heavily under development and is not intended for end-user use. Th
 - A desktop platform which can compile .NET 4.5 (tested on macOS, linux and windows). We recommend using [Visual Studio Code](https://code.visualstudio.com/) (all platforms) or [Visual Studio Community Edition](https://www.visualstudio.com/) (windows only), both of which are free.
 - Make sure you initialise and keep submodules up-to-date.
 
+# Build
+
+### Build on ubuntu with commend line
```
Comment (peppy) [inline README.md]: instructions here should be platform agnostic.

File: README.md [hunk 4]
```diff
@@ -11,6 +11,36 @@ This is still heavily under development and is not intended for end-user use. Th
...
+  
+  In osu directory
+
+```msbuild```
+    
+**5. Launch osu!.exe**
+  
+   In osu directory
+   
+ ```mono osu.Desktop/bin/Debug.osu\!.exe```
```
Comment (peppy) [inline README.md]: this should be platform agnostic, not specifically mono.

**Comment by peppy** [comment]
most of what you add here is classified as "common sense". if you can't figure this out then you probably have no reason to build the project in the first place.

**Comment by DevSDK** [comment]
But, I think we need specification that info somewhere for beginner osu dev.
Developer not only use visual studio and visual studio code.
I'm not commen sense that thing OpenTK require sdl2.
Also global project osu's  'common sense'  write somewhere specification.

Well, Please What Should I do.
Close PR? or Fix some words?

Thx for your review.
